tests/beta/test_default_tensor.py

```diff
@@ -838,7 +838,7 @@ def test_load_tensornet_device(self, rep):
 
         dev = qml.device("default.tensor", wires=2, representation=rep)
         assert dev.num_wires == 2
-        assert dev.shots == 1000
+        assert dev.shots is None
```

Comment:
How come this is changed?

Reply:
Before, the default was `analytic=True, shots=1000`. But now we have to decide between `shots=None` and `shots=1000`. To reproduce the general analytic default, we chose `shots=None`.